models/psnrResultRequester.js

```diff
@@ -0,0 +1,99 @@
+module.exports = class PsnrResultRequester {
```

Comment:
This is the confusing part... This file is actually used by the react app but you're using module.exports which is being used primarily by the node-side files. I'm assuming it all still works because of webpack.